lib/srv/db/postgres/sql/update-permissions.sql

```diff
@@ -0,0 +1,19 @@
+CREATE OR REPLACE PROCEDURE teleport_update_permissions(username VARCHAR, permissions_ JSONB)
```

Comment:
I don't think there are any new requirements here.

Reply:
I've run into errors when activate an user like
```bash
$ tsh db connect --db-name test manual-postgres-auto-drop
psql: error: connection to server at "localhost" (::1), port 52417 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (127.0.0.1), port 52417 failed: your Teleport role requires automatic database user provisioning but an attempt to activate database user "STeve" failed due to the following error: ERROR: permission denied for table pg_statistic (SQLSTATE 42501)
```

I believe this from remove_permissions.sql and I got like a million warnings if I run `call teleport_remove_permissions('STeve');` manually from my "teleport-admin"

Reply:
I'll have a closer look at the minimum permissions required for this to work.

Reply:
I've tested the setup with a non-superuser `teleport-admin` role. I've adjusted the `teleport_remove_permissions` procedure to only go through access rights that the users have, instead of all schemas. Since the user should have received these permissions from `teleport-admin`, it shouldn't fail at removal job either.

aad211da6b5d9a46c3268793f8c2312882f896f2

Reply:
Thanks! Tested and this worked out great with `rds_superuser` role granted to `teleport-admin`.